Objects/longobject.c

```diff
@@ -1248,14 +1247,10 @@ PyLong_AsNativeBytes(PyObject* vv, void* buffer, Py_ssize_t n, int flags)
         /* Calculates the number of bits required for the *absolute* value
          * of v. This does not take sign into account, only magnitude. */
         uint64_t nb = _PyLong_NumBits((PyObject *)v);
-        if (nb == (uint64_t)-1) {
-            res = -1;
-        } else {
-            /* Normally this would be((nb - 1) / 8) + 1 to avoid rounding up
-             * multiples of 8 to the next byte, but we add an implied bit for
-             * the sign and it cancels out. */
-            res = (Py_ssize_t)(nb / 8) + 1;
-        }
+        /* Normally this would be((nb - 1) / 8) + 1 to avoid rounding up
```

Comment:
```suggestion
        /* Normally this would be ((nb - 1) / 8) + 1 to avoid rounding up
```

Reply:
These lines were not changed, except indentation. But well, we can use opportunity to fix formatting.